jobs/e2e_node/crio/crio.ign

```diff
@@ -26,7 +26,7 @@
         "name": "dbus-tools-install.service"
       },
       {
-        "contents": "[Unit]\nDescription=Download and install crio binaries and configurations.\nAfter=network-online.target\n\n[Service]\nType=oneshot\nExecStartPre=/usr/bin/bash -c '/usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -o /usr/local/crio-nodee2e-installer.sh  https://raw.githubusercontent.com/cri-o/cri-o/40cdd9c2d97384eb5601c1af28e7092cdda3815e/scripts/node_e2e_installer; ln -s /usr/bin/runc /usr/local/bin/runc'\nExecStart=/usr/bin/bash /usr/local/crio-nodee2e-installer.sh\n\n[Install]\nWantedBy=multi-user.target\n",
+        "contents": "[Unit]\nDescription=Download and install crio binaries and configurations.\nAfter=network-online.target\n\n[Service]\nType=oneshot\nExecStartPre=/usr/bin/bash -c '/usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -o /usr/local/crio-nodee2e-installer.sh  https://raw.githubusercontent.com/cri-o/cri-o/a560c8d8f1e0b7fdfccec476b87cb070b69c6fb9/scripts/node_e2e_installer'\nExecStart=/usr/bin/bash /usr/local/crio-nodee2e-installer.sh\n\n[Install]\nWantedBy=multi-user.target\n",
```

Comment:
@endocrimes I dropped the ln, the new change should mean it's not needed

Reply:
looks like we'd also need to update  https://github.com/cri-o/cri-o/blob/main/scripts/get#L247-L250 potentially?

Reply:
I don't think so. the get script unconditionally installs to `/usr/local/bin`, both the runc installation and the relabel reference the same location. CRI-O then finds the runc binary in $PATH (either installed in the distro, or installed by the get script), and uses it. Since the get script is homogeneous, and it's CRI-O that is flexible, I think we're good.

Reply:
👍

/approve

(needs a rebase)